crates/libs/bindgen/src/functions.rs

```diff
@@ -50,7 +50,7 @@ fn gen_win_function(gen: &Gen, def: MethodDef) -> TokenStream {
         Some(link) => quote! { #[link(name = #link, kind = "static")] },
         None => {
             if gen.namespace.starts_with("Windows.") {
-                quote! { #[link(name = "windows")] }
+                quote! { #[cfg_attr(any(windows, feature = "link_windows"), link(name = "windows"))] }
```

Comment:
Let's drop the feature for now and baby-step into changes here.

Reply:
Okay, I can do that. No harm in doing it that way either, it simply means that the few functions one may want/need to override on non-Windows can be imported from anywhere instead of specifically (and optionally, because it's a non-default feature) residing in a `libwindows.a`/`libwindows.so` library.